chapter11/sxg/ex_11_2.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+use DateTime;
+
+if ( @ARGV == 1 ){
+	my $now = DateTime−>from_epoch( epoch => time );
+	
+    
+	my $time = DateTime->new(year => $ARGV[0], month => $ARGV[1],day > $ARGV[2]);
```

Comment:
`day > $ARGV[2]` I don't think this will work, I doubt if you tested this, haha :smiley: